### ISSUE TITLE
A request to add proper docstrings to functions.

### ISSUE THREAD

**Issue by teferi** [Suor/funcy#17]
First of — I really like some of the abstractions the library introduces. But exploring the library in python/ipython REPL shell is really hard without proper docstrings.
compare 

```
In [5]: funcy.repeat?
Type:       type
String Form:<type 'itertools.repeat'>
File:       /Users/teferi/git/eksmo/env/lib/python2.7/lib-dynload/itertools.so
Docstring:
repeat(object [,times]) -> create an iterator which returns the object
for the specified number of times.  If not specified, returns the object
endlessly.
```

to

```
In [6]: funcy.retry?
Type:       function
String Form:<function retry at 0x1063a1848>
File:       /Users/teferi/git/eksmo/env/lib/python2.7/site-packages/funcy/flow.py
Definition: funcy.retry(*dargs, **dkwargs)
Docstring:  <no docstring> 
```

Reading the latter really doesn't shed any light on how to use it. I understand that it's a lot of work — to properly document the whole thing, but it would be really nice to have such descriptions.

Also I'm willing to offer some help with that some time later (if that's ok =))

**Comment by Suor** [comment]
Now that we have succint descriptions in cheatsheat it could make sense to copy them to docstrings. Sadly duplication seems inevitable, maybe even prefferable over some custom docs script.

**Comment by Suor** [comment]
Finished in 902e14087a6afe773c6cd534942844e813b85850. To be released in 1.9, soon )
